tomviz/Behaviors.cxx

```diff
@@ -50,6 +55,16 @@ Behaviors::Behaviors(QMainWindow* mainWindow)
   : Superclass(mainWindow)
 {
   Q_ASSERT(mainWindow);
+  vtkSMReaderFactory::AddReaderOfInterest("sources", "JPEGSeriesReader");
+  vtkSMReaderFactory::AddReaderOfInterest("sources", "PNGSeriesReader");
+  vtkSMReaderFactory::AddReaderOfInterest("sources", "TIFFSeriesReader");
+  vtkSMReaderFactory::AddReaderOfInterest("sources", "TVRawImageReader");
+  vtkSMReaderFactory::AddReaderOfInterest("sources", "MRCSeriesReader");
+  vtkSMReaderFactory::AddReaderOfInterest("sources", "CSVReader");
+
+  this->tomvizPlugin = new tomvizExtensions_Plugin();
+  vtkPVPlugin::ImportPlugin(this->tomvizPlugin);
+
```

Comment:
See this [link](http://www.paraview.org/Wiki/ParaView/Plugin_HowTo#Plugins_in_Static_Applications).

You should simply use `PV_PLUGIN_IMPORT_INIT`  and `PV_PLUGIN_IMPORT` macros.